onnxruntime/core/session/inference_session_utils.h

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/session/inference_session.h"
+#include "core/framework/session_options.h"
+#include "core/common/common.h"
+#include "single_include/nlohmann/json.hpp"
+
+using json = nlohmann::json;
+
+namespace onnxruntime {
+
+namespace inference_session_utils {
+
+static const std::string kOrtConfigKey = "ort_config";
```

Comment:
Tiny suggestion: the next time, please try to convert the const strings to "constexpr const char*", instead of std:string. Because the std strings are constructed at runtime, not compile time. and they are global variables. People may run into  [static initialization order ‘fiasco’ (problem)](https://isocpp.org/wiki/faq/ctors#static-init-order)

Reply:
I see. Thanks for the tip. I ll keep this in mind.